quasar/main.tf

```diff
@@ -475,3 +475,12 @@ resource "aws_db_instance" "quasar" {
   performance_insights_enabled    = true
   enabled_cloudwatch_logs_exports = ["postgresql", "upgrade"]
 }
+
+# Testing Bastion SSH dynamic runtime
+module "ssh-bastion-service" {
+  source      = "joshuamkite/ssh-bastion-service/aws"
+  version     = "5.0.0"
+  aws_profile = ""
```

Comment:
Huh… weird that this is required, but can be left blank! 🤔I'm having a hard time understanding what this is supposed to be used for, based on the module's documentation...

Reply:
Yeah, same. The one thing [I think](https://github.com/joshuamkite/terraform-aws-ssh-bastion-service/blob/9a61e7c1f1c228951d5297083c5341b0b7de9260/sts_assumerole_example/policy_example.tpl) it might be is allowing role assumption at a certain point of the process, but it's a bit unclear to me what exactly. 

I'm definitely taking test-as-we go approach, due to lots of working parts stitched together. Reason PR took this long to get in was I was reading/following the docs trying to understand all the components, but I think testing/firing it up will provide more insight in a controlled environment.